### DISCUSSION TITLE
Parseq recognition model has very low confidence scores (but recognized text is good)

### DISCUSSION THREAD

**Discussion by NikitaPomies** [mindee/doctr#1613] [General]
Hello everyone,

I have been using doctr for a while now and I have been very satisfied with it. So thanks for your amazing work ! 

Recently, I wanted to switch from the crnn_vgg16 recognition model to the parseq one.

I noticed that **the confidence scores returned by the parseq model are really low in average ( < 0.4 %)** even when the recognition is perfect.

See an example with this image 
![git_link_test_ocr](https://github.com/mindee/doctr/assets/83075701/dec4f945-42d4-45de-8cae-d1a9ecdb1a8c)

Here a some detected words and their confidence scores returned by parseq : 
![Capture d’écran du 2024-05-28 09-31-45](https://github.com/mindee/doctr/assets/83075701/46de65aa-2a43-406f-8170-e72ce463dfe0)

The recognized text is ok though.

Here is the recognition using crnn_vgg16 : 
![Capture d’écran du 2024-05-28 09-33-07](https://github.com/mindee/doctr/assets/83075701/f3408e1f-9e22-407a-988b-d16af38d26f4)

Does someone know what could be the reason behind that ? Or is it just the model that is badly calibrated ?

**Comment by felixdittrich92** [comment]
Hi @NikitaPomies :wave:,

great to hear you like docTR :smiley: 

Which version of docTR do you use ? I rememeber there was a bug for the `vitstr / parseq` models before 0.8.0

**Comment by NikitaPomies** [comment]
Hi @felixdittrich92 

My bad, I was using version 0.7.0. The pb is indeed corrected in version 0.8.0

**Reply by felixdittrich92** [reply]
:hugs: :+1:
